bigbluebutton-html5/imports/ui/components/end-meeting-confirmation/container.jsx

```diff
@@ -2,17 +2,22 @@ import React from 'react';
 import { withTracker } from 'meteor/react-meteor-data';
 import { withModalMounter } from '/imports/ui/components/modal/service';
 import { makeCall } from '/imports/ui/services/api';
+import Auth from '/imports/ui/services/auth';
+import Users from '/imports/api/users';
 import EndMeetingComponent from './component';
 
 const EndMeetingContainer = props => <EndMeetingComponent {...props} />;
 
+const ROLE_MODERATOR = Meteor.settings.public.user.role_moderator;
+
 export default withModalMounter(withTracker(({ mountModal }) => ({
   closeModal() {
     mountModal(null);
   },
 
   endMeeting: () => {
-    makeCall('endMeeting');
+    const isUserModerator = Users.findOne({ userId: Auth.userID }).role === ROLE_MODERATOR;
```

Comment:
I thought you were going to check inside the method handlers on server side
https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/api/meetings/server/methods/endMeeting.js